urllib3/poolmanager.py

```diff
@@ -112,6 +113,8 @@ class PoolManager(RequestMethods):
 
     def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+        if "transport_security_manager" not in connection_pool_kw:
+            connection_pool_kw["transport_security_manager"] = TransportSecurityManager()
```

Comment:
So I think `transport_security_manager` is something that connection pools have to be keyed off: different transport security managers should cause different connection pools to be created.

Additionally, we should probably store the transport security manager we're using *on* the PoolManager itself: given that it's a global object, the correct way for users to get access to it will be from the pool manager.